src/screens/LoginScreen.js

```diff
@@ -1,14 +1,97 @@
-import { StyleSheet, Text, View } from "react-native";
-import React from "react";
-
-const LoginScreen = () => {
-    return (
-        <View>
-            <Text>LoginScreen</Text>
-        </View>
-    );
+import {
+  StyleSheet,
+  Text,
+  SafeAreaView,
+  Button,
+  TextInput,
+  Alert
+} from "react-native";
+import { firebase } from "../firebase/Config";
+import { useState } from "react";
+
+const initialState = {
+  email: "",
+  password: ""
+};
+
+const LoginScreen = ({ navigation }) => {
+  const [credentials, setCredentials] = useState(initialState);
+
+  function handleChange(event, name) {
+    setCredentials({
+      ...credentials,
+      [name]: event.target.value
+    });
+  }
+
+  console.log(credentials);
+
+  async function handleSubmit() {
+    await firebase
+      .auth()
+      .signInWithEmailAndPassword(credentials.email, credentials.password)
+      .catch((e) => {
+        console.error(e);
+      });
+
+    const currentUser = await firebase.auth().currentUser;
+
+    console.log(currentUser);
+
+    if (currentUser === null) {
+      Alert.alert("Incorrect Email / Password");
+      await firebase.auth().signOut();
+    } else if (!currentUser.emailVerified) {
+      Alert.alert("Please Verify Your Email Before Logging In");
+      await firebase.auth().signOut();
+    } else {
+      navigation.navigate("Dashboard");
+    }
+  }
+
+  return (
+    <SafeAreaView style={styles.Container}>
+      <Text> Log In </Text>
+      <Text> email </Text>
+      <TextInput
+        placeholder="email"
+        value={credentials.email}
+        onChange={(e) => handleChange(e, "email")}
```

Comment:
Should be changed to onChangeText={(text) => handleChange(text, "email")} or something like that